OpenRA.Mods.Common/UpdateRules/Rules/20210321/UnhardcodeBaseBuilderBotModule.cs

```diff
@@ -1,4 +1,4 @@
-﻿#region Copyright & License Information
+#region Copyright & License Information
```

Comment:
This change looks weird, are we adding or removing a BOM here?

Reply:
huh. I edited this file with a text editor as I didn't want to open vscode and it seems it changed some metadate or whatever than it

Reply:
https://en.wikipedia.org/wiki/Byte_order_mark